examples/sample-p5-app/src/serialization.js

```diff
@@ -17,13 +17,41 @@ export const save = function(workspace) {
   window.localStorage?.setItem(storageKey, JSON.stringify(data));
 };
 
+const defaultData = {
+  'blocks': {
+    'languageVersion': 0,
+    'blocks': [
```

Comment:
Do you want these to be non-editable non-deletable non-movable?

Reply:
No, I want them to be movable at the user's convenience. They're marked not deletable in the block's definition. I don't think editable matters because they don't have editable fields.